lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1957,6 +1957,108 @@ def plot(self, xs, ys, *args, zdir='z', **kwargs):
 
     plot3D = plot
 
+    def fill_between(self, x1, y1, z1, x2, y2, z2, *,
+                     where=None, mode='auto', **kwargs):
+        """
+        Fill the area between two 3D curves.
+
+        The curves are defined by the points (*x1*, *y1*, *z1*) and
+        (*x2*, *y2*, *z2*). This creates one or multiple quadrangle
+        polygons that are filled. All points must be the same length N, or a
+        single value to be used for all points.
+
+        Parameters
+        ----------
+        x1, y1, z1 : float or 1D array-like
+            x, y, and z  coordinates of vertices for 1st line.
+
+        x2, y2, z2 : float or 1D array-like
+            x, y, and z coordinates of vertices for 2nd line.
+
+        where : array of bool (length N), optional
+            Define *where* to exclude some regions from being filled. The
+            filled regions are defined by the coordinates ``pts[where]``,
+            for all x, y, and z pts. More precisely, fill between ``pts[i]``
+            and ``pts[i+1]`` if ``where[i] and where[i+1]``. Note that this
+            definition implies that an isolated *True* value between two
+            *False* values in *where* will not result in filling. Both sides of
+            the *True* position remain unfilled due to the adjacent *False*
+            values.
+
+        mode : {'auto', 'quad', 'polygon'}, default: 'auto'
+            The fill mode.
+            If 'auto' and all the coordinates for the x, y, or z input pairs
+            are the same, then the points lie on a plane and 'polygon' is used.
+            Otherwise, 'auto' defaults to 'quad'.
+            If 'quad', then a separate quadrilateral polygon is created for
+            each pair of subsequent points in the 1st and 2nd lines. This is
+            more flexible, but may be slower and result in rendering artifacts.
+            If 'polygon', then the 1st and 2nd lines are connected to form a
+            single polygon, which may be faster and render more cleanly for
+            simple shapes (eg, for filling between two lines that lie on a
+            single plane).
+
+        **kwargs
+            All other keyword arguments are passed on to `.Poly3DCollection`.
+
+        Returns
+        -------
+        `.Poly3DCollection`
+            A `.Poly3DCollection` containing the plotted polygons.
+
+        """
+        _api.check_in_list(['auto', 'quad', 'polygon'], mode=mode)
+
+        had_data = self.has_data()
+        x1, y1, z1, x2, y2, z2 = cbook._broadcast_with_masks(x1, y1, z1, x2, y2, z2)
+        if mode == 'auto':
+            if np.all(x1 == x2) or np.all(y1 == y2) or np.all(z1 == z2):
+                mode = 'polygon'
+            else:
+                mode = 'quad'
+
+        if where is None:
+            where = True
```

Comment:
Could this be the default for the keyword? (you might need to broadcast this out in the above call with the points)

Reply:
The default is `None` in the 2D case and wanted to match that api